sites/fast-website/src/public/style.css

```diff
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Montserrat', sans-serif;

```

Comment:
Do we have a way to ensure Montserrat is available? Will we need to load a web font?

Reply:
ah yes, we don't, will remove. I used it for testing